### PR TITLE
feat(updates-jenkins-io) use an umbrella PVC and Ingress

### PR DESCRIPTION
**Pull request by dduportal**
Related to https://github.com/jenkins-infra/helpdesk/issues/2649

### CODE REVIEW

File: config/updates.jenkins.io.yaml [hunk 1]
```diff
@@ -11,99 +9,71 @@ mirrorbits-lite:
...
     tls:
       - secretName: updates-jenkins-io-tls
         hosts:
           - azure.updates.jenkins.io
 
+  storage:
+    enabled: true
+    claimNameTpl: '{{ include "mirrorbits-parent.pvc-name" $ }}'
+    storageClassName: azurefile-csi-premium
+    storageSize: 2Gi # See file share size in https://github.com/jenkins-infra/azure/blob/main/updates.jenkins.io.tf
```
Comment (lemeurherve) [inline config/updates.jenkins.io.yaml:25]:
```suggestion
    storageSize: 2Gi  # See file share size in https://github.com/jenkins-infra/azure/blob/main/updates.jenkins.io.tf
```
Fix "too few spaces before comment" yamllint warning

File: config/updates.jenkins.io.yaml [hunk 2]
```diff
@@ -11,99 +9,71 @@ mirrorbits-lite:
...
+          persistentVolumeReclaimPolicy: Retain
+          mountOptions:
+            - dir_mode=0755
+            - file_mode=0644
+            - uid=1000
+            - gid=1000
+            - mfsymlinks
+            - nobrl
+            - serverino
+            - cache=strict
```
Comment (lemeurherve) [inline config/updates.jenkins.io.yaml:43]:
```suggestion
        persistentVolumeReclaimPolicy: Retain
        mountOptions:
          - dir_mode=0755
          - file_mode=0644
          - uid=1000
          - gid=1000
          - mfsymlinks
          - nobrl
          - serverino
          - cache=strict
```
Fix "wrong indentation: expected 8 but found 10" yamllint error

**Review by lemeurherve — APPROVED** [review]
Compared the before/after in the diff, the only notable change is `readOnly: true`, justified as the accessMode is set to `ReadOnlyMany` anyway, LGTM!

Excellent job on these charts 👏

**Comment by dduportal** [comment]
Thanks for the indentation fix and the review!

Let's roll

**Comment by dduportal** [comment]
Deployment failed with the following error:

```
Error: UPGRADE FAILED: an error occurred while rolling back the release. original upgrade error: failed to create resource: admission webhook "validate.nginx.ingress.kubernetes.io" denied the request: host "azure.updates.jenkins.io" and path "/" is already defined in ingress updates-jenkins-io/updates-jenkins-io-mirrorbits-lite: no Secret with the name "updates-jenkins-io-mirrorbits-lite-binary" found
```

WiP

**Comment by dduportal** [comment]
Had to delete the ingress and the former PVC. It's deployed.

I can see the files served by httpd, but can't reach mirrorbits: gotta check the ingress

**Pull request merged**